modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -1,45 +1,38 @@
 // Module included in the following assemblies:
 //
-// * virt/live_migration/virt-configuring-live-migration-policies.adoc
-
+// * virt/live_migration/virt-configuring-live-migration.adoc
 
 :_content-type: PROCEDURE
 [id="virt-configuring-a-live-migration-policy_{context}"]
-= Configuring a live migration policy from the command line
+= Creating a live migration policy by using the command line
 
-Use the `MigrationPolicy` custom resource definition (CRD) to define migration policies for one or more groups of selected virtual machine instances (VMIs).
+You can create a live migration policy by using the command line.
 
-You can specify groups of VMIs by using any combination of the following:
+A live migration policy is applied to selected virtual machines (VMs) by using any combination of labels:
 
-* Virtual machine instance labels such as `size`, `os`, `gpu`, and other VMI labels.
-* Namespace labels such as `priority`, `bandwidth`, `hpc-workload`, and other namespace labels.
+* VM labels such as `size`, `os`, or `gpu`.
+* Project labels such as `priority`, `bandwidth`, or `hpc-workload`.
```

Comment:
Just to make sure, did you get QE verification for these types of changes? I know there is technically a difference between VMIs and VMs, projects and namespaces... my understanding is that they are _often_ interchangeable, but they aren't always. I do remember the original story being specifically about VMIs.

Reply:
I asked Stu Gott and he said it was OK.